x/staking/keeper/keeper.go

```diff
@@ -44,6 +44,7 @@ type Keeper struct {
 	Redelegations               collections.Map[collections.Triple[[]byte, []byte, []byte], types.Redelegation]
 	Delegations                 collections.Map[collections.Pair[sdk.AccAddress, sdk.ValAddress], types.Delegation]
 	UnbondingIndex              collections.Map[uint64, []byte]
+	Validators                  collections.Map[sdk.ValAddress, []byte]
```

Comment:
```suggestion
	Validators                  collections.Map[[]byte, []byte]
```
is this possible?

Reply:
we could use bytes, but that wouldn't be state compatible because validator address is a generic address which has its length prepended. In order to these settle compatibility issues, we use `LengthPrefixedAddressKey` for ValAddress.

Reply:
We'd need to check using DiffCollectionsMigration but I think it should be compatible if we use `LengthPrefixedBytesKey`